specs/engage-digital_openapi3.yaml

```diff
@@ -7978,6 +8029,109 @@ components:
+        created_at:
+          format: date-time
+          type: string
+        updated_at:
+          format: date-time
+          type: string
+        name:
+          description: |-
+            Name displayed in ED admin interface.
+            Is not displayed to end users.
+          type: string
+        link:
+          description: |-
+            Link to the survey on the provider website.
+          type: string
+        active:
+          type: boolean
+        from_name:
+          description: |-
+            Name displayed as sender when survey is sent to end user.
+            Currently only applies to surveys sent by email.
+          type: string
+        instant_send_threshold:
+          description: |-
+            All intervention shorter than this value (in seconds) will have the survey sent instantly.
+          type: number
+        max_time_since_last_reply:
+          description: |-
+            If last message from end user is older than this value (in seconds), no survey will be sent.
+          type: number
+        send_delay:
+          description: |-
+            Delay before sending survey to end users.
+          type: number
+        send_message:
+          type: string
+        last_activated_at:
+          description: |-
+            Internal field.
+            Updated each time the survey is enabled on the ED admin interface.
+          format: date-time
+          type: string
+        last_response_at:
+          description: |-
+            Internal field.
+            Time of last survey response. Used for polling responses from provider.
+          format: date-time
+          type: string
+        category_ids:
+          items:
+            type: string
+          type: array
+        team_ids:
+          items:
+            type: string
+          type: array
+        source_ids:
+          items:
+            type: string
+          type: array
+        type:
+          description: |-
+            Contains the information of the survey provider used.
+            Currently, only provider supported is alchemer.
+          enum: ['alchemer']
+          type: string
+        questions:
+          description: |-
+            Fetched from survey provider.
+            List of questions asked in this survey.
+          items:
+            $ref: '#/components/schemas/SurveyQuestion'
+          type: array
+    SurveyQuestion:
+      properties:
+        foreign_id:
+          description: |-
+            The survey provider's id for this question.
+          type: number
+        main:
+          type: boolean
+        text:
+          description: |-
+            The question's body.
+          type: string
+        choices:
+          description: |-
+            Fetched from survey provider.
+            List of choices end user can elect to answer this question.
+          items:
+            $ref: '#/components/schemas/SurveyQuestionChoices'
+    SurveyQuestionChoices:

```

Comment:
I could have reused the `SurveyQuestionReply` below instead of having a new model but I thought it would bring confusion to someone editing the doc as it does not reflect the architecture of our code.

It is debatable.

Reply:
ok like that